comp120-evaluation.tex

```diff
@@ -0,0 +1,58 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\usepackage{graphicx}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{A Reflection on how my Code Comprehension, Time Management and Communication Skills have Affected my Learning}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1507290}
+
+\begin{document}
+
+\maketitle
+
+\abstract{}
+
+\section{Introduction}
+I have identified three key weaknesses that need addressing: understanding peers' code, time management and communication.
+
+\section{Reading and Understanding Peers' Code}
+Due to a lack of experience in reading and writing code, I am experiencing difficulty comprehending peers' code. Despite being able to understand small sections, I struggle to see how it all fits together. I immediately noticed that this makes constructing useful comments in peer review sessions difficult.
+
```

Comment:
These paragraphs are all quite short and could be merged as they each correspond to a common theme/idea.